jsonnet/components/alertmanager.libsonnet

```diff
@@ -22,6 +22,29 @@ function(params)
         namespace: cfg.namespace,
       },
       spec: {
+        path: '/api/v1',
```

Comment:
Is there a need to have the /api/v1 path exposed?

Reply:
Thanks for pointing out. 
Openshift console is not using v1 endpoint. Not sure if other components use v1.
I will update this PR limiting the route to v2 and see what happens in the tests.